app/src/main/java/com/nishan/moviedb/DownloadImage.java

```diff
@@ -0,0 +1,47 @@
+package com.nishan.moviedb;
+
+
+import android.graphics.Bitmap;
+import android.graphics.BitmapFactory;
+import android.graphics.drawable.Drawable;
+import android.os.AsyncTask;
+import android.widget.ImageView;
+import android.widget.RelativeLayout;
+
+import java.io.InputStream;
+
+public class DownloadImage extends AsyncTask<String, Void, Bitmap> {
+    ImageView imageView;
+    RelativeLayout layout = null;
+
+    public DownloadImage(ImageView imageView) {
+        this.imageView = imageView;
+    }
+
+    public DownloadImage(ImageView imageView, RelativeLayout layout) {
+        this.imageView = imageView;
+        this.layout = layout;
+    }
+
+    protected Bitmap doInBackground(String... urls) {
+        String imageURL = urls[0];
+        Bitmap bimage = null;
+        try {
+            InputStream in = new java.net.URL(imageURL).openStream();
+            bimage = BitmapFactory.decodeStream(in);
+
+        } catch (Exception e) {
```

Comment:
Do not handle generic exceptions. You should check only MalformedURLException here.